wso2-synapse-unit-test-plugin/src/main/java/org/wso2/synapse/unittest/SynapseTestCaseFileReader.java

```diff
@@ -361,6 +361,10 @@ private static void processConnectorResourcesData(OMElement artifactsNode) throw
 
             String encodedConnectorFile;
             if (!resource.getText().isEmpty()) {
+                if (!resource.getText().endsWith(".zip")) {
+                    String connectorZipPath = Paths.get("target", "dependency", resource.getText() + ".zip").toString();
```

Comment:
Shall we have constants for these strings?

Reply:
Fixed in 023353187910928dc7c43012473b0c08b5a75e2c